src/eth/primitives/execution_conflict.rs

```diff
@@ -58,4 +67,16 @@ pub enum ExecutionConflict {
         expected: SlotValue,
         actual: SlotValue,
     },
+
+    Account {
+        address: Address,
+        expected_nonce: BigDecimal,
+        expected_balance: BigDecimal,
+    },
+
+    PgSlot {
+        address: Address,
+        slot: Vec<u8>,
+        expected: Vec<u8>,
```

Comment:
why not use [u8; 32] if it's gonna be converted later?

Reply:
True